sa/rate_limits.go

```diff
@@ -58,30 +58,44 @@ func (ssa *SQLStorageAuthority) addCertificatesPerName(db db.SelectExecer, names
 // countCertificates returns the count of certificates issued for a domain's
 // eTLD+1 (aka base domain), during a given time range. It assumes that the
 // given db already has a context associated with it.
-func (ssa *SQLStorageAuthority) countCertificates(dbMap db.Selector, domain string, timeRange *sapb.Range) (int64, error) {
-	var counts []int64
+func (ssa *SQLStorageAuthority) countCertificates(dbMap db.Selector, domain string, timeRange *sapb.Range) (int64, *time.Time, error) {
```

Comment:
Why do you use `*time.Time` and return `nil` in the error case here, while using `time.Time` and returning `time.Time{}` in the error case in the RA?

You can use https://pkg.go.dev/time#Time.IsZero to determine if a time.Time is completely uninitialized, just like you're using a nil check in the caller of this function.

I don't have an incredibly strong opinion whether we use `time.Time` and `IsZero` or `*time.Time` and nil checks (though I slightly prefer the former), but we should be consistent.

Reply:
That's fair. I've gone ahead and refactored to the former.